pkg/storage/unified/resource/storage_backend.go

```diff
@@ -56,6 +56,11 @@ func NewKvStorageBackend(kv KV) *kvStorageBackend {
 	}
 }
 
+func (k *kvStorageBackend) ListModifiedSince(ctx context.Context, key ResourceModifiedKey, sinceRv int64, cb func(iterator ListIterator) error) (int64, error) {
```

Comment:
@chaudyg I was going to leave this implementation for another PR since I dont think we need this on the kv store impl right now?

Reply:
Yep, I'll leave it for now and make a ticket